### ISSUE TITLE
datamatrix doesn't compile on kinetic

### ISSUE THREAD

**Issue by marc-hanheide** [strands-project/strands_social#50]
see https://lcas.lincoln.ac.uk/buildfarm/job/Kdev__strands_social__ubuntu_xenial_amd64/3/console:
```
/tmp/catkin_workspace/src/strands_social/datamatrix_read/src/data_matrix.cpp:49:5: error: ‘imwrite’ is not a member of ‘cv’
```

@Jailander can you quickly check what is wrong with this. Probably some simple opencv header issue.

**Comment by Jailander** [comment]
HI just saw this will fix tomorrow morning, although one wonders is there any sense on maintaining this package? we are not using it anymore (since 2014 I think) and we have better options.

**Comment by marc-hanheide** [comment]
Yes, that's the other option. Moved it to a deprecated branch and remove here. Maybe even better!

**Comment by Jailander** [comment]
Ok I will do that :)

**Comment by Jailander** [comment]
I have created a PR (#51 ) removing the package, and created a deprecated branch on the main repo with the previous package
